### ISSUE TITLE
Client-side performance

### ISSUE THREAD

**Issue by joelkuiper** [drugis/mcda-elicitation-web#40]
We should start to worry about the performance of the (angular.js) client-side. Especially the overview page is slow due to a large amount of bindings. 

We could check-out https://github.com/Pasvaz/bindonce for some of the bindings.

**Comment by joelkuiper** [comment]
I've started using the angular 1.3 bind-once functionality {{::var}} vs {{var}} and got rid of the old Foundation javascript (in favor of the pure angular directives). This has sped up the client-side a bit. We should still be watching the number of $watchers though, there is a lot going on that doesn't need to be that complicated. 

Another thing that has sped up the app a bit is getting rid of the calls to R in the controllers, instead doing them only once when we construct the workspace. This prevents at least one digest cycle.

**Comment by joelkuiper** [comment]
Switching MathJax to SVG also made sense

**Comment by joelkuiper** [comment]
We also deferred the angular bootstrap till domReady, this was unnecessary.
If you remove the ng-app declaration in HTML you can simply call angular.bootstrap and it will work fine. 
See  https://github.com/drugis/mcda-elicitation-web/blob/develop/app-remote/src/main/webapp/resources/app/js/main.js#L38
